xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -738,6 +803,49 @@ bool CGUIWindowPVRGuideBase::OnContextButtonDate()
   return bReturn;
 }
 
+bool CGUIWindowPVRGuideBase::Go12HoursBack()
+{
+  return GotoDate(-12);
+}
+
+bool CGUIWindowPVRGuideBase::Go12HoursForward()
+{
+  return GotoDate(+12);
+}
+
+bool CGUIWindowPVRGuideBase::GotoDate(int deltaHours)
+{
+  CGUIEPGGridContainer* epgGridContainer = GetGridControl();
+  const CDateTime date = deltaHours < 0
+                       ? epgGridContainer->GetSelectedDate() - CDateTimeSpan(0, std::abs(deltaHours), 0, 0)
+                       : epgGridContainer->GetSelectedDate() + CDateTimeSpan(0, deltaHours, 0, 0);
```

Comment:
Aren't both of these the same as `CDateTimeSpan(0, std::abs(deltaHours), 0, 0)` now?

Reply:
Haha, ofc. Good catch.